CRBM-Viz/src/app/Layout/links/links.component.ts

```diff
@@ -8,5 +9,13 @@ import { AuthService } from 'src/app/Services/auth0.service';
 export class LinksComponent implements OnInit {
   constructor(public auth: AuthService) {}
 
-  ngOnInit() {}
+  ngOnInit() {
```

Comment:
Is this meant to be added to this component? This component is intended to list out the various pages that we have on the site, So I don't think it needs to know anything about the authentication. If we want to show the user profile I think that would be best in another component

Reply:
This is meant to be added to any component which is initialized at very start. This seemed to me a good place to put it. What do you think?